doc/requirements.txt

```diff
@@ -4,8 +4,8 @@ jupyter
 matplotlib>=3
 nbsphinx==0.5.1
 pandas>=1
+sphinx>=2.4.4,<3.1
 sphinx-autodoc-typehints>=1.10.3
 sphinx-copybutton
-sphinx>=2.4.4,<3.1
```

Comment:
why do this dependency reorderings keep happening?

Reply:
See commit message of 6fd3ba19f3a6886c0444649ed3f80d83145408b5